server/schemas/wave.py

```diff
@@ -23,23 +20,23 @@ class Historical_Wave_Base(BaseModel):
 
 
 # Properties to receive on item creation
-class Historical_Wave_Create(Historical_Wave_Base):
+class WaveCreate(WaveBase):
     ...
 
 
 # Properties to receive on item update
-class Historical_Wave_Update(Historical_Wave_Base):
+class WaveUpdate(WaveBase):
     ...
 
 
 # Properties shared by models stored in DB
-class Historical_Wave_InDBBase(Historical_Wave_Base):
+class WaveInDBBase(WaveBase):
```

Comment:
Add a class that extends from `WaveInDBBase`. This represents properties to return to client. See `Coastline` in schemas/coastline.py for reference.

Reply:
Not fully sure what to do here, since coastlines returns a EWKT object. While these models only return texts. I still have to run some tests as this most recent commit I pushed up just to save my work and continue working on later. Still have to test running server with DB connection.

Reply:
For the schema files, you're just missing the class that represents what the client will be receiving.

```python
class WaveDatum(WaveDatumInDBBase):
    # make sure the relevant fields are selected
    # will be easier to visualize once you start testing
    # so you can stub it out like so in the meantime
    ...
```

These files are our DTO with validation. Each class will serve a purpose as documented in those comments.

Reply:
I added this [hyperlink](https://github.com/clairBuoyant/server/blob/main/server/schemas/coastline.py#L34-L35) to in my [original comment](https://github.com/clairBuoyant/server/pull/44#discussion_r945210700). This points to the Coastline class that I was referencing. Hope this helps clarify any confusion. Let me know if you have any questions.

Reply:
Schema files are closely linked to our model files. It's almost duplicative. They serve different purposes though.

The files in models reflect what our DB schema looks like. 

The files in schemas contain our DTOs. Some of which reflect what our DB schema looks like (those will have "DB" in the class name) and others will reflect what the client (e.g., frontend application) will receive.

Say we had a User model. We don't want to send sensitive information like passwords over to the frontend system. So we wouldn't define that particular class with the password attribute. We use `pydantic` for validation of our schemas/DTOs, so it will ensure it is not included as part of the payload by either raising an error (for us to fix) or parsing it out.

Reply:
Ok it makes sense now. I just left them with `...` but I would like some reading material so that it could make total sense to me. I think I get why in coastlines it had to return a specific ewkt object because the client can't understand binary data. The ellipses mean just return the same exact type of data to the client.

Reply:
> The ellipses mean just return the same exact type of data to the client.

Not exactly. Since we're composing this with inheritance, the classes you create will share whatever properties and functionalities were defined in the parent class.

The `...` is equivalent to [`pass`](https://www.w3schools.com/python/ref_keyword_pass.asp). It's like a placeholder without adding any functionality.